### PR TITLE
Adds a new quirk, sleeping beauty

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Creates a new quirk for the game, known as the sleeping beauty. Whenever you are put to sleep with this quirk, you will *deathgasp. When people examine you while asleep, they will see you as dead. CMO is an exception, and can always tell whether you are truly dead. All huds, machinery, medibots, etc are unaffected as well.

## Why It's Good For The Game
Interesting dynamic, indirectly buffs all sleep inducers, everyone outside of medical has to be very careful to make sure you're still alive,  basically a death wish if you take phobia

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

sb_menu
sb_std
sb_lyzer

Demonstrates the cmo's immunity to being fooled:
sb_cmo

</details>

## Changelog
:cl:
add: adds a new quirk, sleeping beauty, which makes sleeping appear as being dead
/:cl:

### CODE REVIEW

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -947,11 +949,11 @@
 
 /datum/status_effect/slimegrub
 	id = "grub_infection"
-	duration = 60 SECONDS //a redgrub infestation in a slime
```
Comment (etherware-novice) [inline code/datums/status_effects/debuffs.dm:950]: have no idea why these are marked with a change

**Comment by 12Oreos** [comment]
Seems interesting, a good way to get the morgue. Does this also happen when you go into crit? because it would fool a would be murderer and could be seen as a potential buff

**Comment by etherware-novice** [comment]
not 100% sure where the crit code is, so no not yet

**Comment by FortSumter** [comment]
IDK how I feel about this. More positive traits are good as the incentivize people to actually take traits. This one's a bit funky though as there isn't any IRL reasoning behind why this would be a thing, and you might have a lot of staff that might incorrectly identify you as a ling or otherwise dump you in places such as the morgue where you can't escape. You take a massive risk by "dying"  like this.

Other problems of note is that you can use the IC menu to go to sleep whenever you like, you can usually be woken up and forced to stand if some tries to shake you awake, and it would be quite odd if the "dead" body started to snore....

**Comment by etherware-novice** [comment]
i disabled snoring if you have the quirk

File: code/datums/status_effects/debuffs.dm (hunk 1 above)
Comment (itsmeow) [reply code/datums/status_effects/debuffs.dm:950]: removing a trailing space. it's part of your auto-formatter. it's fine.

**Comment by PowerfulBacon** [comment]
This just doesn't make sense to me IC-wise and could encourage ignoring bodies since they are probably just sleeping.

**Comment by etherware-novice** [comment]
> This just doesn't make sense to me IC-wise and could encourage ignoring bodies since they are probably just sleeping.

thats the POINT, its a NEGATIVE QUIRK, and 90% of people probably wont take it

**Comment by FortSumter** [comment]
I'd consider this a positive perk as playing dead in certain situations can be an absolute win. I don't agree with Bacons reasoning for why its bad though. Given the current meta of parameds showing up to murders in progress and random crewmembers getting themselves involved whenever a valid does anything bad I think it'd be a nice change if people left bodies alone.

**Comment by etherware-novice** [comment]
the fact youre taking it is a risk, so therefore if your body gets ignored THATS ON YOU

**Comment by PowerfulBacon** [comment]
It's a negative quirk that can result in the effects going onto other players rather than just affecting yourself.
This just doesn't make enough sense to me IC and feels like a shallow quirk for a free 4 bonus quirk points.
Closing for above reasons

**Pull request closed without merging**